Source/Csla/DataPortalClient/HttpProxyOptions.cs

```diff
@@ -36,6 +36,16 @@ public class HttpProxyOptions
     /// </summary>
     public Func<IServiceProvider, HttpClient> HttpClientFactory { get; set; } = DefaultHttpFactory;
 
+    /// <summary>
+    /// Gets or sets the timeout value for the HTTP request.
+    /// </summary>
+    public TimeSpan Timeout { get; set; }
+
+    /// <summary>
+    /// Gets or sets the read-write timeout value for the HTTP request.
+    /// </summary>
+    public TimeSpan ReadWriteTimeout { get; set; }
```

Comment:
Shouldn't we state that this only applies to sync http calls and not async ones with HttpClient?

Reply:
I didn't get this part of the async in the docs, [https://learn.microsoft.com/en-us/dotnet/api/system.net.httpwebrequest.readwritetimeout?view=net-8.0&redirectedfrom=MSDN#System_Net_HttpWebRequest_ReadWriteTimeout](https://learn.microsoft.com/en-us/dotnet/api/system.net.httpwebrequest.readwritetimeout?view=net-8.0&redirectedfrom=MSDN#System_Net_HttpWebRequest_ReadWriteTimeout)

Reply:
We use `WebClient` _only_ for sync calls. For async calls we use `HttpClient`. So this timeout only applies to sync calls in CSLA and I think we should state that to avoid misunderstanding. Or am I misunderstand your comment here? 😅

Reply:
I thought that in .net framework this timeout wasn't used in async calls, and I didn't find any comment about that